config/config.yaml

```diff
@@ -249,4 +249,10 @@ assets:
             - dist/hotkeys.min.js
     reason-code-widget:
         basePath: '%webroot%/library/js/'
-        script: reasonCodeWidget.js
\ No newline at end of file
+        script: reasonCodeWidget.js
+    erx:
+        basePath: '%webroot%/library/js/'
+        script: erx_javascript.js
+    checkpwd_validation:
+        basePath: '%webroot%/interface/usergroup/'
+        script: checkpwd_validation.js
```

Comment:
just noted this checkpwd_validation, which doesn't look like is being used anywhere

Reply:
Yea, I plan on using it in a subsequent commit.